pkg/config/store.go

```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"strings"
+	"sync/atomic"
+
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+)
+
+type Config struct {
+	EnabledFormats []string
+}
+
+const (
+	enabledFormatsKey = "formats.enabled-formats"
+)
+
+func parse(data map[string]string) Config {
+	cfg := Config{}
+	if formats, ok := data[enabledFormatsKey]; ok {
```

Comment:
Do we want an `else` here to catch any failures, something like the code example for  "comma, ok" idiom https://golang.org/doc/effective_go.html#interface_conversions

Reply:
I don't think we need one here - that would just indicate the user left this as a default value, which would be equivalent to an empty string or no enabled formats.

I added a comment explaining why we need the check at all - unfortunately strings.Split is a little tricky here.